tox.ini

```diff
@@ -42,29 +31,19 @@ commands =
 [testenv:w_zodb]
 basepython =
     python2.7
-commands =
-    python setup.py -q test -q
 deps =
-    zope.interface
-    persistent
-    transaction
+	{[testenv]deps}
```

Comment:
Indentation seems a tad excessive.  Is this a literal tab?

Reply:
Whoops, I think it was.